src/dragonball/src/lib.rs

```diff
@@ -32,7 +32,7 @@ pub mod vcpu;
 /// Virtual machine manager for virtual machines.
 pub mod vm;
 
-mod event_manager;
+pub mod event_manager;
```

Comment:
Why does this switch to `pub`?
Don't know if I miss something, but I couldn't find where you use event_manager in thie pull request.

Reply:
`event_manager` is previously used in a CLI parser. After removing unused components, I found this modification unnecessary. So I will reset the access policy.